utils/dataset.py

```diff
@@ -463,14 +477,27 @@ def __init__(self, args, path, config):
         )
 
         # depth parameters
-        self.has_depth = False
-        self.depth_scale = None
+        self.has_depth = True
+        self.num_frames = 0
+        self.accumulated_time = 0.0
+        self.previous_frame_time = time.time()
+        #self.depth_scale = None
 
     def __getitem__(self, idx):
         pose = torch.eye(4, device=self.device, dtype=self.dtype)
 
         frameset = self.pipeline.wait_for_frames()
-        rgb_frame = frameset.get_color_frame()
+        aligned_frames = self.align.process(frameset)
+        
+        current_frame_time = time.time()
+        self.num_frames = self.num_frames + 1
+        if self.num_frames > 1:
+            time_diff = (current_frame_time - self.previous_frame_time)
+            self.accumulated_time = self.accumulated_time + time_diff
+            if self.num_frames%50 == 0:
+                print("Inst and Avg FPS", 1/time_diff, self.num_frames/self.accumulated_time)
```

Comment:
Remove these lines as the timing measurement is redundant.

Reply:
Removed